ingest/profiles/default/config.yaml

```diff
@@ -0,0 +1,4 @@
+cores: all
```

Comment:
Changed from `1` to `all` but I think that's fine.

Reply:
I don't know why we ever used 1 in the first place